### ISSUE TITLE
Add field for  initials of team member completing Collection Data Entry- Clinical Collection (Biospecimen) lower priority

### ISSUE THREAD

**Issue by erincschwartz** [episphere/connect#1114]
On the Collection Data Entry screen in the Research Dashboard, there is a field for Collection Phlebotomist Initials (concept ID 719427591). 

![Image](https://github.com/user-attachments/assets/8d4116a3-a2d2-4a2c-b937-a7850aedd7b4)

Please add the same  field to the Collection Data Entry screen in the Clinical Dashboard. This field can be placed in the equivalent location, beneath Collection ID Link Date/Time. 
Instead of "Collection Phlebotomist Initials", please title this field "**Initials of team member completing Collection Data Entry**" 

![Image](https://github.com/user-attachments/assets/78c7ff1e-7a09-460f-95f9-9cde02ced7af)

The definition of this variable in the Data Dictionary will need to be broadened to include the new context for Clincial Collections.

**Comment by sonyekere** [comment]
Similar to #1012

**Comment by nbirus** [comment]
@erincschwartz I think I have a handle on this ticket, I do need to clarify the final statement: 

> The definition of this variable in the Data Dictionary will need to be broadened...

I'm not sure what or where the Data Dictionary lives.  Is this something I can update, or is there someone I can ask for help?

**Comment by brotzmanmj** [comment]
Hi Nate, please consult with @mnataraj92 , added to this issue, thanks

**Comment by mnataraj92** [comment]
Hi @nbirus please let me know what changes are needed, I can add them to the dictionary change log, and once Nicole approves, I can add it to the dictionary. Is it just that the variable name and label "Collection Phlebotomist Initials" should be changed to "Initials of team member completing Collection Data Entry"?

**Comment by nbirus** [comment]
Thanks @mnataraj92.  To be honest I'm not quite sure what the data change log is, so I don't know what changes are needed.  I don't think we are replacing the "Collection Phlebotomist Initials", but adding a new field to capture the initials of a team member for the clinician's workflow.   I see in the UI we have a map of key/values we are using to pass data to the backened e.g. `phlebotomistInitials: 719427591`, is this related?  Do we need to create a new one of these for this new field?

**Comment by mnataraj92** [comment]
@nbirus @erincschwartz Hi Erin, are we looking to create a new variable for "Initials of team member completing Collection Data Entry", or did we want to incorporate this into the already existing "Collection Phlebotomist Initials" and just change the variable name/variable label to incorporate the initials of a team member?

If we want a new variable, would it basically be set up the same way as Collection Phlebotomist Initials (concept ID 719427591), but it would just be called Initials of team member completing Collection Data Entry?

**Comment by erincschwartz** [comment]
We want to incorporate this into the already existing "Collection Phlebotomist Initials" variable and update the variable name/variable label to reflect both uses. Current question text could be updated to say: "Initials of Research Collection Phlebotomist/Team Member Completing Clincial Collection Data Entry". Note can be updated to say "For Research Collections, initials of the individual who completed blood collection are entered. For Clincal Collections, initials of the team member completing the collection entry into the Biospecimen Dashboard are entered. This free-text box is a required field to continue to the review page for both Research and Clinical collections."

@nbirus Please make "Initials of team member completing Collection Data Entry" a required field to continue to the review page for Clinical Collections (programming similar to what what you are currently working on for the Research Collection phlebotomist initials)

**Comment by mnataraj92** [comment]
Thanks Erin! I'll add this to the data dictionary change log

**Comment by nbirus** [comment]
Sounds good, I will use the same `719427591` code for both workflows, and I will update the label to the above.  I will also try a few different implementations of the note and post some screenshots here for review.  Thanks!

**Comment by erincschwartz** [comment]
@nbirus The note is for the Data Dictionary, not to be posted in the Dashboard. Let me know if you have questions about this.

**Comment by nbirus** [comment]
In dev and ready to test

**Comment by mnataraj92** [comment]
Just to make a note that this variable has been updated in the dictionary

**Comment by erincschwartz** [comment]
**Tested in Dev, ready for Stage**

- Checked to make sure that initials must be entered before Collection Data Entry screen could be saved. Received error message, as expected
![Image](https://github.com/user-attachments/assets/82261e94-15b0-40cc-8025-d73d127d012f)

- Saved and returned to the collection to ensure initial were retained, worked as expected. 
- ![Image](https://github.com/user-attachments/assets/d73ac41f-a2cb-41b7-b775-8783428496cb)

**Comment by erincschwartz** [comment]
During Stage testing I realized that the _Initials of Team Member Completing Clincial Collection Data Entry_ are not present on the Review Collection Data Entry screen, like the initials field is for Research Collections, see below. The _Initials of Team Member Completing Clincial Collection Data Entry_ should appear where the red arrow is in the CLINICAL example. My apologies that I did not realize this during Dev testing. 

**RESEARCH:** 
![Image](https://github.com/user-attachments/assets/d9ce0565-72ae-4b82-8b6f-ea42e154355e)

**CLINICAL:**
![Image](https://github.com/user-attachments/assets/42bc4f0c-239f-4d6d-81fb-4c32bb9302ef)







The other functionality for adding this initials field worked as expected: 
- Checked to make sure that initials must be entered before Collection Data Entry screen could be saved. Received error message, as expected
![Image](https://github.com/user-attachments/assets/b54ac211-f6db-49c7-bb24-0f324be1676e)

- Saved and returned to the collection to ensure initial were retained, worked as expected.
![Image](https://github.com/user-attachments/assets/2a0e83d6-b81b-4279-bc2d-2e72222558ee)
